kong/db/schema/others/declarative_config.lua

```diff
@@ -37,6 +39,12 @@ local foreign_references = {}
 local foreign_children = {}
 
 
+local function now_updated()
+  update_time()
```

Comment:
Do we really need it to be this accurate? IMO, `ngx.req.start_time()` is good enough.
Plus, what if a field is about to expire and a request accesses this field multiple times?

Reply:
`ngx.time()` without `ngx.update_time` should be more than accurate enough for this purpose.

Reply:
done